probe/endpoint/reporter.go

```diff
@@ -157,6 +158,10 @@ func (r *Reporter) Report() (report.Report, error) {
 				fromNodeInfo[report.HostNodeID] = hostNodeID
 			}
 
+			if conn.Proc.NetNamespaceID > 0 {
```

Comment:
Good point. They are inodes, I am not sure of an inode with value 0 is acceptable, but we can make this conditional to the PID being non-zero.